bravado/fido_client.py

```diff
@@ -119,7 +116,8 @@ def prepare_request_for_twisted(request_params):
             # using requests < 2.8.1 due to a bug while handling unicode values
             # See changelog 2.8.1 at https://pypi.python.org/pypi/requests
             'method': str(prepared_request.method or 'GET'),
-            'body': prepared_request.body,
+            'body': to_bytes(prepared_request.body) if prepared_request.body \
```

Comment:
I am not a huge fan of the line return here, can you rewrite this with parenthesis rather ?
```
(
    to_bytes(prepared_request.body) if prepared_request.body
    if prepared_request.body is not None
    else None
)
```
That's really accessory though